src/detours.cpp

```diff
@@ -974,6 +974,19 @@ inline PBYTE detour_skip_jmp(PBYTE pbCode, PVOID *ppGlobals)
     return pbCode;
 }
 
+inline void detour_find_jmp_bounds(PBYTE pbCode,
```

Comment:
I wonder if we shoudn't have common code:
`detour_find_jmp_within_2gb` or `detour_find_jmp_within_2gb_above_or_below` or `detour_find_jmp_boudnds_common`?

Reply:
I think Pedro's PR adds what you have here?

Reply:
Typo in commit message (sorry).